assets/templates/homepage/census-tile.tmpl

```diff
@@ -8,7 +8,6 @@
                     <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-white-landscape-cy.svg" alt="{{ localise "Census2021" .Language 1 }}" class="header__svg-logo"
                     xmlns="http://www.w3.org/2000/svg" focusable="false" width="209" height="40" viewBox="0 0 228 44">
```

Comment:
I think it would be better practice to make this a self closing tag (same for the image tag on lines 5-6) as you have correctly removed the closing tag:

```suggestion
                    <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-white-landscape-cy.svg" alt="{{ localise "Census2021" .Language 1 }}" class="header__svg-logo"
                    xmlns="http://www.w3.org/2000/svg" focusable="false" width="209" height="40" viewBox="0 0 228 44"/>
```